src/components/CredentialCard/ShareCredentialCard.tsx

```diff
@@ -33,6 +33,7 @@ interface Props {
   onSelect: (value: Selected) => void;
   viewRef: RefObject<HTMLElement>;
   isSelected?: boolean;
+  isLastOfFew: boolean;
```

Comment:
I feel that the name choice here makes the card aware of too many details. I believe the card has no need to know it’s last of few, it only should know whether it should be expanded or not. This is why its sibling CredentialCard has a prop called `expand`.